lib/ansible/modules/remote_management/ucs/ucs_mac_pool.py

```diff
@@ -32,7 +32,7 @@
     - "You cannot use spaces or any special characters other than - (hyphen), \"_\" (underscore), : (colon), and . (period)."
     - You cannot change this name after the MAC pool is created.
     required: yes
-  descrption:
```

Comment:
I wonder if you should create an alias: descrption 
So that existing playbooks are not broken.   And maybe it would fix the validation error.